src/parsers/xccdf.ts

```diff
@@ -118,8 +160,12 @@ export function processXCCDF(xml: string, removeNewlines: false, useRuleId: 'gro
       default:
         throw new Error('useRuleId must be one of "group", "rule", or "version"')
     }
-        
-    control.title = removeXMLSpecialCharacters(rule['@_severity'] ? ensureDecodedXMLStringValue(rule.title, 'undefined title') : `[[[MISSING SEVERITY FROM BENCHMARK]]] ${ensureDecodedXMLStringValue(rule.title,'undefined title')}`)
+
+    if (!(_.isArray(rule.title) && rule.title.length === 1)) {
+      throw new Error('Rule title is not an array of length 1.');
```

Comment:
Again, I am not sure what is best to say, but what can we say to the reader to give them some action to understand what to do if they receive this error.

```suggestion
      throw new Error('Rule title is not an array of length 1. Investigate if the file is in the proper format.');
```